### PR TITLE
Add get board for project

### PR DESCRIPTION
**Pull request by liweihu1**

### CODE REVIEW

File: src/services/BoardService.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
 /* eslint-disable no-new, no-restricted-syntax, no-await-in-loop */
+const axios = require( "axios" );
+const { pickBy } = require( "lodash" );
 const { MessageConsumer, MessageProducer } = require( "../config/messaging/Kafka" );
 const Column = require( "../models/Column" );
 const Board = require( "../models/Board" );
 const Task = require( "../models/Task" );
 
+const ProjectURI = process.env.PROJECT_MANAGEMENT_URI;
```
Comment (tomdewildt) [inline src/services/BoardService.js]: Move this to the getFinishedBoardTasks function and rename to uri

File: src/controllers/BoardController.js [hunk 2]
```diff
@@ -13,6 +13,15 @@ const getBoards = async ( req, res ) => {
     return res.json( boards );
 };
 
+const getBoardsForProject = async ( req, res ) => {
```
Comment (tomdewildt) [inline src/controllers/BoardController.js]: Forgot to add JSDoc

File: src/routes/BoardRoutes.js [hunk 3]
```diff
@@ -35,6 +35,29 @@ const router = express.Router();
  */
 router.get( "/", asyncMiddleware( controller.getBoards ) );
 
+/**
+ * /boards/for/:
```
Comment (tomdewildt) [inline src/routes/BoardRoutes.js]: Forgot ```@swagger``` tag

File: src/controllers/BoardController.js (hunk 2 above)
Comment (drokii) [reply src/controllers/BoardController.js]: added jsdoc

File: src/routes/BoardRoutes.js (hunk 3 above)
Comment (drokii) [reply src/routes/BoardRoutes.js]: added tag

**Pull request merged**